Competition2017/tests/competition/subsystems/collector/MockCollectorSubsystem.java

```diff
@@ -0,0 +1,30 @@
+package competition.subsystems.collector;
+
+import com.google.inject.Inject;
+
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class MockCollectorSubsystem extends CollectorSubsystem {
+    
+    @Inject
+    public MockCollectorSubsystem(WPIFactory factory, XPropertyManager propManager) {
+        super(factory, propManager);
+        // TODO Auto-generated constructor stub
+        
+    }
+    public XCANTalon getMotor(){
+        return collectorMotor;
+    }
+    
+    public DoubleProperty getIntake(){
```

Comment:
Can we append the word `Prop` or `Property` to make it clear what the purpose is?